gl_extensions/src/main.cpp

```diff
@@ -0,0 +1,200 @@
+#include <thread>
```

Comment:
Either remove this example, or make it conform to the coding standrts

Reply:
Removed it